unstructured/__version__.py

```diff
@@ -1 +1 @@
-__version__ = "0.16.21"  # pragma: no cover
+__version__ = "0.16.22-dev0"  # pragma: no cover
```

Comment:
Does this have to be a release (not dev release) in order to publish a new image to the quay repo?